### DISCUSSION TITLE
채팅봇 관련 문의드립니다!

### DISCUSSION THREAD

**Discussion by hikaru-hi** [Lastorder-DC/chatreader-kor#36] [문의] [answered]
안녕하세요 채팅봇 감사드립니다.
다름이 아니라 제가 일본에서 방송을하며 사용을하고있는데 읽어주는 속도가 너무빨라 이해하지못하는 경우가 종종생기더군요 ㅠㅠ 혹시 속도부분을 조절할수있는 방법이 있나 문의드립니다

**Comment by Lastorder-DC** [comment, marked as answer]
속도 조절은 아직 불가능하며 `!!tts uniq off`로 개인화 설정을 끄시면 기본 속도가 사용되므로 조금 느려질 수 있습니다.
